### ISSUE TITLE
Summary

### ISSUE THREAD

**Issue by Tooconfident** [SkylarkingMonkey/f1-3-c2p1-colmar-academy#5]
Grade: Meets Expectations

Solid job on this project. You show a solid understanding of responsive design concepts and with a little practice you should have it down. Check my line notes to revisit some choices where the transitions to different screen sizes get a little choppy. There are a lot do media query values out there so it is hard to follow at times, but if you just keep playing with the code and working with adjusting your screen, you will see begin to see where the issues are and what to address. Using chrome dev tools worked well with me when I inspected your page to find the transition issues, so feel free to use that to your advantage. Great job overall, the website looks beautiful and your max and min windows look flawless. Well done!

**Comment by SkylarkingMonkey** [comment]
Hey Ryan,  

   Thanks a bunch for looking over my project.  I really appreciate it.  To be honest I wanted to put more time into it, but I'm happy to report that I've been tight on time because I've been working on my first freelance gig, so I don't feel super guilty about my capstone project getting sidelined..

Cheers,

Jean-Marc
